frappe/core/doctype/link_field_filter/link_field_filter.json

```diff
@@ -0,0 +1,53 @@
+{
+ "actions": [],
+ "allow_rename": 1,
+ "creation": "2023-10-05 17:07:50.711810",
+ "default_view": "List",
+ "doctype": "DocType",
+ "editable_grid": 1,
+ "engine": "InnoDB",
+ "field_order": [
+  "link_field",
+  "field",
+  "condition",
+  "value"
+ ],
+ "fields": [
+  {
+   "fieldname": "link_field",
+   "fieldtype": "Select",
+   "in_list_view": 1,
+   "label": "Link Field"
+  },
+  {
+   "fieldname": "field",
+   "fieldtype": "Select",
+   "in_list_view": 1,
+   "label": "Field"
+  },
+  {
+   "fieldname": "value",
+   "fieldtype": "Data",
+   "in_list_view": 1,
+   "label": "Value"
+  },
+  {
+   "fieldname": "condition",
+   "fieldtype": "Select",
+   "in_list_view": 1,
+   "label": "Condition"
```

Comment:
I think, this should be called "Operator".
The condition is all three taken together.